.github/workflows/deploy-dev.yml

```diff
@@ -4,7 +4,7 @@ on:
   workflow_dispatch:
   push:
     branches:
-      - dev
+      - DAS-518-device-data-endpoints
```

Comment:
It should remain dev, no?

Reply:
This change is how the deploy to dev happens.  Since I am not merging it, I am leaving it.